server/index.js

```diff
@@ -14,30 +16,69 @@ app.use(express.json());
 
 app.use('/items/:item_id', express.static('react-client/dist'));
 
-app.get('/item/images', (req, res)=>{
-  pool.query('SELECT * FROM image_urls', (err, data) => {
-    if (err){
+
+
+app.get('/item/:item_id/images', (req, res)=> {
+  const id = req.params.item_id;
+  controller.images.getOne(id)
+    .then((response) => {
+      console.log(response)
+      res.send(response);
+    })
+    .catch((err) => {
       res.status(400).send(err);
-    } else {
-      res.status(200).send(data)
-    }
-  });
+    });
 });
 
+app.post('/item/:item_id/images', (req, res)=>{
+  const id = req.body.item_id;
+  const url = req.body.url;
+  controller.images.post(id, url)
+    .then((response) => {
+      console.log(response.data)
```

Comment:
This may be more of a personal preference, but sometimes specific comments in the console.logs can help find where the log is coming from quicker. E.g. "server post response: ${response.data}"

Reply:
oh thanks! I agree with you that more descriptive, the better. I actually think i forgot to remove this altogether so thanks for pointing it out.